Makefile

```diff
@@ -22,7 +22,7 @@ check: lint fmt vet test
 build: ## Build binary
 	@echo -e "\033[32mBuilding package...\033[0m"
 	mkdir -p bin
-	$(DOCKER_CMD) go build -v -o bin/machine-api-operator cmd/main.go
+	$(DOCKER_CMD) go build -v -o bin/machine-api-operator cmd/machine-api-operator/*.go
```

Comment:
No need to specify any `*.go` file if you use the full path as Andrew suggested.